crates/rooch/src/commands/account/commands/create.rs

```diff
@@ -46,31 +47,42 @@ use std::{
 pub struct CreateCommand {
     #[clap(flatten)]
     pub context_options: WalletContextOptions,
+    /// Command line input of crypto schemes (0 for Ed25519, 1 for MultiEd25519, 2 for Ecdsa, 3 for Schnorr)
+    #[clap(short = 's', long = "scheme")]
+    pub crypto_schemes: String,
 }
 
 impl CreateCommand {
     pub async fn execute(self) -> RoochResult<ExecuteTransactionResponseView> {
         let mut context = self.context_options.build().await?;
-        let (new_address, phrase, scheme) = context
-            .config
-            .keystore
-            .generate_and_add_new_key(Ed25519, None, None)?;
+        match BuiltinScheme::from_flag(self.crypto_schemes.clone().trim()) {
+            Ok(scheme) => {
+                let (new_address, phrase, scheme) = context
+                    .config
+                    .keystore
+                    .generate_and_add_new_key(scheme, None, None)?;
 
-        println!("{}", AccountAddress::from(new_address).to_hex_literal());
-        println!(
-            "Generated new keypair for address with scheme {:?} [{new_address}]",
-            scheme.to_string()
-        );
-        println!("Secret Recovery Phrase : [{phrase}]");
+                println!("{}", AccountAddress::from(new_address).to_hex_literal());
+                println!(
+                    "Generated new keypair for address with scheme {:?} [{new_address}]",
+                    scheme.to_string()
+                );
+                println!("Secret Recovery Phrase : [{phrase}]");
 
-        let create_account_entry_function = CREATE_ACCOUNT_ENTRY_FUNCTION.clone();
-        let action = MoveAction::new_function_call(
-            create_account_entry_function,
-            vec![],
-            vec![bcs::to_bytes(&new_address).unwrap()],
-        );
+                let create_account_entry_function = CREATE_ACCOUNT_ENTRY_FUNCTION.clone();
+                let action = MoveAction::new_function_call(
+                    create_account_entry_function,
+                    vec![],
+                    vec![bcs::to_bytes(&new_address).unwrap()],
+                );
 
-        context.sign_and_execute(new_address, action).await
+                context.sign_and_execute(new_address, action).await
```

Comment:
The `sign_and_execute` should use the new scheme and private_key to sign the transaction

Reply:
Got it.